src/map/index.es

```diff
@@ -6,16 +6,19 @@ import classNames from 'classnames';
 import { reverse } from '../utils';
 import { getMapOptions, getTileUrl, getTileOptions, isViewChanged } from './utils';
 
+import { Provider } from './context';
+
 
 class Map extends PureComponent {
   constructor(props) {
     super(props);
 
     this.elements = [];
     this.state = {
-      isReady: false,
+      isContextReady: false,
     };
 
+    this.staticContext = this.getDefaultContext();
```

Comment:
Lets call it `mapContext` and assign an empty object `{}` in constructor. Initial object is unusable anyway